From 6d23d7b91388213ff090159f52af3aa88a6684c8 Mon Sep 17 00:00:00 2001
From: Allain Legacy <allain.legacy@windriver.com>
Date: Mon, 26 Feb 2018 08:04:47 -0600
Subject: [PATCH 2/2] spec: include ipv6 fixes from 2.4 release candidate

Pulling in patches from the 2.4 release candidate series in order to address
various IPv6 formatting/parsing issues.

Signed-off-by: Allain Legacy <allain.legacy@windriver.com>
---
 SPECS/scapy.spec | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/SPECS/scapy.spec b/SPECS/scapy.spec
index add5a84..7afe35f 100644
--- a/SPECS/scapy.spec
+++ b/SPECS/scapy.spec
@@ -15,6 +15,7 @@ URL: http://www.secdev.org/projects/scapy/
 # https://bitbucket.org/secdev/scapy/pull-request/80
 Source0: https://github.com/%{gituser}/%{gitname}/archive/v%{version}.tar.gz#/%{name}-%{version}.tar.gz
 Patch0: scapy-2.2.0-xdg-open.patch
+Patch1: 0001-Use-socket.pton-socket.ntop-is-available.patch
 
 BuildArch: noarch
 BuildRequires: python >= 2.5
@@ -31,6 +32,7 @@ requests and replies, and much more.
 %prep
 %setup -q
 %patch0 -p1
+%patch1 -p1
 
 %build
 %{__python} setup.py build
-- 
2.12.1

